int8 total_hp
time game_start_time
time game_end_time
PlayerState[] players
